### PR TITLE
Merge quality check steps together

### PR DESCRIPTION
**Pull request by theofidry**
I would like to simplify the "quality check" process locally and instead of having multiple commands to run:

- `make cs`
- `make analyze` which also offers the choice of the two subsets
  - `make phpstan`
  - `make validate`
- `make test-autoreview`

Instead I'm proposing `make autoreview` which does:

- `phpstan`
- `validate`
- `test-autoreview`

In other words, `make analyze`, `make cs` is kept as a separate step for now (maybe worth integrating there? I couldn't really make up my mind here). The others are **NOT** removed, simply hidden from the default help list.

### CODE REVIEW

**Comment by maks-rafalko** [comment]
It appears that `make autoreview` and `make analyze` are now partly duplicated, which does not make sense (at least for now). Do we need them both?

Also, I would leave the name `analyze` rather than `autoreview` because many projects use the same name, so it makes sense to continue using it here. Wdyt?

**Comment by theofidry** [comment]
`analyze` has actually been removed here, but you can still call `phpstan` or `validate`. They are simply not exposed in the help but those targets still exists.

>Also, I would leave the name analyze rather than autoreview because many projects use this name, so it makes sense to continue using it here. Wdyt?

I personally find `autoreview` more explicit than `analyze` but if you prefer it I can change it. Out of curiosity what other projects use `analyze`?

**Comment by maks-rafalko** [comment]
> Out of curiosity what other projects use `analyze`?

At the moment of writing, I was 100% sure that at least PHPStan / PHP-CS-Fixer use that, but it appears they not, can't find anything about it 🤦‍♂ 

What I see right now - I've been using `analyze` for the last 3 years at the company I'm working for, this is what affected my mind I guess. Btw, I constantly mix up `make cs` with `cs:fix` for the same reason - this is how that command is named on our projects.

TL;DR if you guys think `autoreview` is better - ok, but if you don't mind, I would leave it as `analyze` just to not think about it after switching from the job's projects.

**Comment by theofidry** [comment]
I know the trouble of switching :) I'm always mixing up `master` and `develop` for my rebase & merges haha
